build/Dockerfile.tracerunner

```diff
@@ -2,7 +2,8 @@ ARG bpftraceversion=v0.11.1
 FROM quay.io/iovisor/bpftrace:$bpftraceversion as bpftrace
 
 FROM golang:1.11.4-stretch as gobuilder
-
+ARG GIT_ORG=iovisor
```

Comment:
perhaps keep all the args at the top of the file?

Reply:
Technically it is at the top, the args I think have a weird scoping where they belong to the most recent "FROM" statement. The prior one at the top is because it is needed *before* a from statement.

So this one is at the top of its own sub-dockerfile